book/website/community-handbook/coworking.md

```diff
@@ -0,0 +1,26 @@
+# Community Coworking Calls
+
+The concept of coworking is generally associated with the practice of sharing physical office space with different people who work independently such as the remote employees or freelancers.
```

Comment:
Remove "the" from before remote employees, or "the remote employees of an organisation with an office elsewhere"